bpf/http_sock.h

```diff
@@ -189,6 +198,18 @@ static __always_inline void read_skb_bytes(const void *skb, u32 offset, unsigned
     }
 }
 
+// empty_http_info zeroes and return the unique percpu copy in the map
+// this function assumes that a given thread is not trying to use many
+// instances at the same time
+static __always_inline http_info_t* empty_http_info() {
```

Comment:
I moved this here so we can use bpf_memset. memset from the stdlibrary is not available for me in my toolchain and for some reason (I'm not sure why) I couldn't use bpf_memset in http_types.h, some issue with the includes. I had to use `__bpf_memset` and I'd rather stick to the public helpers.

Reply:
Thanks! Strange that actually bpf_memset did not work correctly for me, when running "make docker-generate".